bundle.js

```diff
@@ -136,27 +136,27 @@ const addRequest = (event, bundle, convertedBundle) => {
 };
 
 const addResponse = (event, bundle, convertedBundle) => {
-  if (event.name.endsWith('.post') || event.name.endsWith('.output')) {
```

Comment:
Removed as this seems to be a mistake. Full custom scripting methods, i.e., event names ending with `.output`, don't have a `.response` in their `bundle`.

Reply:
If tests pass 🤷‍♂️ . I'd only double check this isn't for some search output fields stuff, where we'd need the result from the response there.

Reply:
I removed it because it was breaking action custom fields scripting. Not sure if I did it right, but I double checked by reading `bundle.response.content` in a `KEY_custom_search_result_fields`, which doesn't work as `bundle.response` is `undefined`, so I think it should be safe to remove. 🤞